src/components/Home/Home.js

```diff
@@ -0,0 +1,22 @@
+import React, { useState } from "react";
+import { useSelector } from "react-redux";
```

Comment:
treba odvojiti grupe importa
```suggestion
import { useSelector } from "react-redux";

```